proc/fcidump.py

```diff
@@ -0,0 +1,460 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2019 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+"""Module with utility function for dumping the Hamiltonian to file in FCIDUMP format."""
+
+from datetime import datetime
+
+import numpy as np
+
+from psi4.driver import psifiles as psif
+from psi4.driver.p4util.testing import compare_integers, compare_values, compare_recursive
+from psi4.driver.procrouting.proc_util import check_iwl_file_from_scf_type
+
+from psi4 import core
+
+
+def fcidump(wfn, fname='INTDUMP', oe_ints=None, write_pntgrp=False):
+    """Save integrals to file in FCIDUMP format as defined in Comp. Phys. Commun. 54 75 (1989)
+    Additional one-electron integrals, including orbital energies, can also be saved.
+    This latter format can be used with the HANDE QMC code but is not standard.
+    This function converts the irrep labels from Cotton's order (used in psi4)
+    to molpro ordering. Since the latter is ambiguous unless the point group is
+    specificied, this function has an option to write the point group label in the
+    FCIDUMP file. This, however, is not part of the standard.
+
+    :returns: None
+
+    :raises: ValidationError when SCF wavefunction is not RHF
```

Comment:
Is this still valid?

Reply:
No, this needs to be revised in forte and psi4. Thanks for noticing it.